app/views/krikri/harvest_sources/index.html.erb

```diff
@@ -21,5 +26,6 @@
 
 <%= link_to 'New',
     new_institution_harvest_source_path(institution_id: @institution.id)
-%>
+%> | 
```

Comment:
@AudreyAltman :+1: OK, no sweat. I'll close this and resubmit the PR with just one commit for the generator.